napari/layers/transforms.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import annotations  # noqa: F407
```

Comment:
@jni it looks from our tests that this line is not python 3.6 compatible

Reply:
Oops! =) Should we drop 3.6? Note that this would only be slightly ahead of schedule, as NEP-29 has the community dropping 3.6 this June...

Reply:
Note that without this, it is an error to write `-> STTransform:` as the type annotation of a method of `STTransform`. See [PEP-563](https://www.python.org/dev/peps/pep-0563/)

Reply:
I'm also find holding back on those sorts of annotations and keeping 3.6 support till June. Can discuss at team meeting - curious how @kne42 feels